tools/online_editor/src/other.ts

```diff
@@ -0,0 +1,397 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
```

Comment:
This looks like dead (old) code to me. Is it imported anywhere?

Reply:
Oh, right! I saved a copy of the original to be able to look things up and compare and totally forgot about it later:-)